examples/ct/ct_handlers.go

```diff
@@ -251,9 +251,9 @@ func addChainInternal(w http.ResponseWriter, r *http.Request, c RequestHandlers,
 
 	request := trillian.QueueLeavesRequest{LogId: c.logID, Leaves: []*trillian.LeafProto{&leafProto}}
 
-	ctx, _ := context.WithDeadline(context.Background(), getRPCDeadlineTime(c))
-
+	ctx, cancelFunc := context.WithDeadline(context.Background(), getRPCDeadlineTime(c))
 	response, err := c.rpcClient.QueueLeaves(ctx, &request)
+	cancelFunc()
```

Comment:
Not sure what the standard paradigm is here, but various examples seem to prefer "defer cancelFunc()" a line higher - thoughts?